NEWS.md

```diff
@@ -33,6 +33,7 @@
 * `seq_linter()` recommends `rev()` in the lint message for lints like `nrow(x):1` (#1542, @MichaelChirico).
 * New `xp_call_name()` helper to facilitate writing custom linters (#2023, @MichaelChirico). This helper converts a matched XPath to the R function to which it corresponds. This is useful for including the "offending" function in the lint's message.
 * `function_argument_linter()` detects usage of `missing()` for the linted argument (#1546, @MichaelChirico). The simplest fix for `function_argument_linter()` lints is typically to set that argument to `NULL` by default, in which case it's usually preferable to update function logic checking `missing()` to check `is.null()` instead.
+* Allow to opt out of linting `filter()` expressions in `conjunct_test_linter` via new `allow_filter` argument which defaults to `FALSE` (#2110, @salim-b)
```

Comment:
sorry, could you merge it with the existing item on L30? since the inclusion of filter() has not been released yet, I think it makes more sense to have only one item.

Reply:
Ok, see https://github.com/r-lib/lintr/pull/2110/commits/3638b4adba954cadfdf9b3d6267d80fa620628f3